.github/workflows/_ci.yaml

```diff
@@ -205,6 +205,21 @@ jobs:
     with:
       IMAGE: ${{ needs.build-jax.outputs.DOCKER_TAG_FINAL }}
       TEST_NAME: jax
+      EXECUTE: |
+        docker run --shm-size=1g --gpus all ${{ needs.build-jax.outputs.DOCKER_TAG_FINAL }} test-jax.sh -b backend-independent | tee test-backend-independent.log;
+        docker run --shm-size=1g --gpus all ${{ needs.build-jax.outputs.DOCKER_TAG_FINAL }} test-jax.sh -b gpu | tee test-gpu.log
+      STATISTICS_SCRIPT: |
+        errors=$(cat test-*.log | grep -c 'ERROR:' || true);
+        failed_tests=$(cat test-*.log | grep -c 'FAILED in' || true);
+        passed_tests=$(cat test-*.log | grep -c 'PASSED in' || true);
+        total_tests=$((failed_tests + passed_tests));
+        echo "TOTAL_TESTS=${total_tests}" >> $GITHUB_OUTPUT;
+        echo "ERRORS=${errors}" >> $GITHUB_OUTPUT;
+        echo "PASSED_TESTS=${passed_tests}" >> $GITHUB_OUTPUT;
+        echo "FAILED_TESTS=${failed_tests}" >> $GITHUB_OUTPUT;
+      ARTIFACTS: |
+        test-backend-independent.log
```

Comment:
Are sitrep/badge jsons uploaded as artifacts?

Reply:
Yes, sitrep/badge is a common files to upload, so I moved them to `_test_unit.yaml`.